docs/framework/reflection-and-codedom/dynamic-language-runtime-overview.md

```diff
@@ -46,7 +46,7 @@ Examples of languages developed by using the DLR include the following:
 
 - IronPython. Available as open-source software from the [GitHub](https://github.com/IronLanguages/ironpython2) Web site.
 
-- IronRuby. Available as open-source software from the [IronRuby](http://ironruby.net/) website.
+- IronRuby. Available as open-source software from the [IronRuby](https://ironruby.net/) website.
```

Comment:
While I understand the need to prevent warnings, this site shouldn't be linked to if the `https` version doesn't have its certs updated. Navigate to this site:

- https://ironruby.net

I'd argue that instead, we remove this link.